teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)

```

Comment:
- [ ] Please kindly add `specialization` into the `initialize` 🙃 
- [ ] Also in super you don't need to write ` parent_permission: parent_permission ` you just need to write `parent_permission` as you did for` name` and `age` 😁.